packages/notifier/README.md

```diff
@@ -51,36 +63,33 @@ publication.updateState('a');
 publication.updateState('b');
 publication.finish('done');
 ```
-The publisher has published an interation with the non-final sequence
-`'a'`, `'b'`, terminated with `'done'` as its completion value.
 
-Alice the subscriber consumes using the `for-of-await` loop
+Alice, the subscriber, consumes the iteration using the `for-await-of` loop. She can see the 
+non-final values and whether the iteration completes or fails. She can see a failure reason, 
+but the `for-await-of` syntax does not let her see the completion value `'done'`. While she 
+can write code that only executes after the loop finishes, the code won’t know if the completion 
+value was “done”, “completed”, or something else. This is a limitation of JavaScript's iteration, 
+whether asynchronous or synchronous (as consumed by a `for-of` loop).
+
 ```js
 const consume = async subscription => {
   try {
     for await (const val of subscription) {
       console.log('non-final', val);
```

Comment:
```suggestion
      console.log('non-final ', val);
```

Reply:
The standard behavior of `console.log` and all similar console methods is that they put the extra spaces in themselves. @michaelfig actually added code to our `details` template literal tag to remove spaces from the literal text that console would put back in, avoiding two spaces. With your change here, you would likewise get two spaces you did not intend.

Reply:
Yeah, I got confused because I'm nearly always using quasi-quote now. I have to keep upgrading my deeply ingrained habits.